fiftyone/utils/sam.py

```diff
@@ -252,16 +263,19 @@ def _forward_pass_boxes(self, imgs):
                 device=sam_predictor.device,
             )
 
-            masks, _, _ = sam_predictor.predict_torch(
-                point_coords=None,
-                point_labels=None,
-                boxes=transformed_boxes,
-                multimask_output=False,
-            )
+            try:
```

Comment:
Is there a specific reason you're gracefully ignoring errors here? Generally the error should be allowed to be raised, so that the user can decide how it should be handled via `dataset.apply_model(..., skip_failures=)`.